### PR TITLE
Core/PacketIO: Updated and Enabled currently supported loot packets.

### PR DESCRIPTION
**Pull request by Zerixx**
Updated and enabled most of the currently supported loot packets, so that... well they actually work in game. Was questing and this got annoying when I could not loot.

Feedback where necessary always appreciated!

-Z

### CODE REVIEW

File: src/server/game/Server/Packets/LootPackets.cpp [hunk 1]
```diff
@@ -34,7 +34,7 @@ WorldPacket const* WorldPackets::Loot::LootResponse::Write()
     _worldPacket << uint32(Items.size());
     _worldPacket << uint32(Currencies.size());
 
-    for (LootItemData const& item : Items)
+    for (auto const& item : Items)
```
Comment (DDuarte) [inline src/server/game/Server/Packets/LootPackets.cpp]: You don't have to change this but avoid doing these sort of replacements.
`auto` works great for some types (e.g iterators) but it removes a lot of readability from the code

**Comment by DDuarte** [comment]
LGTM

**Comment by msoky** [comment]
what about this AoE looting? looting from multiple corpses around player at once? https://github.com/TrinityCore/TrinityCore/pull/13817

**Comment by Zerixx** [comment]
@msoky It's only updates and enabling of currently supported packets. I don't know anything about AE Looting. May wanna look into it yourself.

**Comment by Shauren** [comment]
I updated item push result (your structure was wrong), please merge/rebase

**Comment by Zerixx** [comment]
I accidentally merged the wrong local branches, and fucked it. Will open new PR.

**Comment by DDuarte** [comment]
@Zerixx: `git config --global user.email yourrealemail@prettyplease.com`

**Comment by Rushor** [comment]
would be awesome if you could do this @Zerixx :)

**Pull request closed without merging**